webapp/components/wpt-flags.html

```diff
@@ -228,16 +229,22 @@ <h4>Server-side only features</h4>
         Allow /api/runs?pr=[GitHub PR number]
       </paper-checkbox>
     </paper-item>
-    <paper-item>
+    <h5>GitHub Status Checks</h5>
+    <paper-item sub-item>
       <paper-checkbox checked="{{failChecksOnRegression}}">
         Fail the wpt.fyi GitHub status check if regressions are found, and pass them if not.
       </paper-checkbox>
     </paper-item>
-    <paper-item>
+    <paper-item sub-item>
       <paper-checkbox checked="{{checksAllUsers}}">
         Run the wpt.fyi GitHub status check for all users, not just whitelisted ones.
       </paper-checkbox>
     </paper-item>
+    <paper-item sub-item>
+      <paper-checkbox checked="{{pendingChecks}}">
+        Create pending GitHub status check when results first arrive, and are being processed.
```

Comment:
Ah, I didn't realize that's when we create it, I assumed we create it even earlier. With this enabled, the time the check is pending should then be a measure of how long the queuing time is. It should be possible to bring that down to just a few seconds, right?

Reply:
When enabled, it's the time it takes for the processor to create a summary and call us back with a POST to /api/results/create. This can be as slow as ~5min, depending on the size of the report.

Reply:
Oh, limited by the time it takes to upload split results to GCS I guess. Getting rid of those files would be sweet :)